source/Cosmos.Debug.HyperVServer/Cosmos.Debug.HyperVServer.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net48</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
```

Comment:
Does this work under windows still? IIRC there was some reason to keep it on net48

Reply:
we can even use hyperV on linux so... if it does break it would be safe to revert and just add build conditions to not include this on linux